helpers/mailSender.js

```diff
@@ -0,0 +1,22 @@
+import { createTransport } from "nodemailer";
+
+export default async function notifyByEmail(receiver, message) {
+    let transporter = createTransport({
+        host: "smtp.gmail.com",
+        port: 465,
+        secure: true, // true for 465, false for other ports
+        auth: {
+            user: "", //set email
+            pass: "", //set password
```

Comment:
Si mal no entiendo, este sería el mail con el cual la librería se loguearía para notificar al usuario que hay stock.

No se si valdría poner los comentaarios con un TODO.
```suggestion
            user: "", // TODO: set email
            pass: "", // TODO: set password
```

Y/o agregar al README que es importante para usar el script agregar el mail o contraseña coon el cual se quiere notificar.

No sé qué opina @matigastirami

Reply:
Si, tendría que ser con TODO y aclarar en el README eso

Reply:
@ezeallio hay una lib en npm que se llama config que te permite agregar un archivo config.json dentro de un directorio /config. Con esta podés hacer algo así en tus archivos:

import config from "config";

...
user: config.get("EMAIL_USER")
...

También está la lib dotenv que me gusta más, pero eso ya va más en gustos.

Por otro lado deberíamos ver como hacer el handling de esto porque cada usuario que la use va a tener un mail diferente y en caso de poner todo acá estaríamos exponiendo data de usuarios.